bin/dune_init.ml

```diff
@@ -524,10 +524,41 @@ module Component = struct
             Path.exists (Path.relative dir ".ocamlformat")
             && Path.exists (Path.relative dir ".gitignore")
           in
+          let fmt_version =
+            try
+              let ic = Unix.open_process_in "ocamlformat --version" in
+              let version = input_line ic in
+              let _ = Unix.close_process_in ic in
+              (let open Pp.O in
+               Console.print_user_message
+                 (User_message.make
+                    [ Pp.tag User_message.Style.Hint (Pp.verbatim "Format")
+                      ++ Pp.textf ": using ocamlversion number %s" version
+                    ]));
+              version
+            with
+            | _ ->
+              (let open Pp.O in
+               Console.print_user_message
+                 (User_message.make
+                    [ Pp.tag User_message.Style.Warning (Pp.verbatim "Format")
+                      ++ Pp.textf ": couldn't get the ocamlformat version number."
+                    ]);
+               Console.print_user_message
+                 (User_message.make
+                    [ Pp.tag User_message.Style.Hint (Pp.verbatim "Format")
+                      ++ Pp.textf ": Are you sure you have ocamlformat installed ?"
+                    ]));
+              ""
+          in
           if exists
```

Comment:
Should I keep this close or not then ? What do you think ? Removing it make `dune` display a nice warning when files already exists in the target directory (when using `dune init proj .` for example)